### ISSUE TITLE
Bug: Object at index 0 is null

### ISSUE THREAD

**Issue by VirsabiMunk** [Deadcows/MyBox#84]
I've been getting a lot of these errors when entering playmode for quite some time now (even before version 1.3). 

I'm having a hard time realizing why it happens, but i think its some kind of forbidden usage of one of the attributes which works in editor but not during playmode. 

```
SerializedObjectNotCreatableException: Object at index 0 is null
UnityEditor.Editor.CreateSerializedObject () (at <cf7d2c2e83f749e8943578126f9eff4b>:0)
UnityEditor.Editor.GetSerializedObjectInternal () (at <cf7d2c2e83f749e8943578126f9eff4b>:0)
UnityEditor.Editor.get_serializedObject () (at <cf7d2c2e83f749e8943578126f9eff4b>:0)
MyBox.Internal.UnityObjectEditor.OnEnable () (at Library/PackageCache/com.mybox@a244358ff4/Tools/Internal/UnityObjectEditor.cs:15)
```

**Comment by Deadcows** [comment]
Sorry for the late response. 
Can you please tell me, did you used several Inspector windows when this happened?

**Comment by Deadcows** [comment]
This is probably [fixed in 2019.1](https://issuetracker.unity3d.com/issues/argumentexception-object-at-index-0-is-null-thrown-when-entering-play-mode-with-two-inspector-windows-and-an-attached-script)
Please let me know if you're using 2019.1+ version and issue persists

**Comment by Deadcows** [comment]
[Found](https://github.com/Deadcows/MyBox/commit/9cec62085ae2d7855f7710d13d1e552bc8305a6b)! It's really strange, but sometimes with Custom Editor for Object type editors created for... null objects. Probably such objects are created by engine for some temp things and destroyed before OnEnable, but OnEnable is called anyway. 

null check for target fixes the thing. I hope it's gone for good
